crates/cdk-config/src/telemetry.rs

```diff
@@ -1,14 +1,13 @@
-use std::net::SocketAddr;
-
-use serde::Deserialize;
-
 use super::DEFAULT_IP;
+use serde::Deserialize;
+use std::net::SocketAddr;
 
 #[derive(Deserialize, Debug, Clone, Copy)]
 #[serde(rename_all = "PascalCase")]
+#[allow(dead_code)]
 pub struct TelemetryConfig {
     #[serde(rename = "PrometheusAddr", default = "default_metrics_api_addr")]
-    pub addr: SocketAddr,
+    pub addr: SocketAddr, //
```

Comment:
Missing comment? Should we remove the slashes?
```suggestion
    pub addr: SocketAddr,
```